### PR TITLE
Add support for PDFs uploaded with type "auto"

### PR DESCRIPTION
**Pull request by TisButMe**

### CODE REVIEW

File: lib/cloudini/url.ex [hunk 1]
```diff
@@ -20,6 +20,19 @@ defmodule Cloudini.URL do
         else
           image_url
         end
+      [head, name, "raw", op, rest] when op in ["upload", "fetch"] ->
+        if List.last(String.split(rest, ".")) == "pdf" do
```
Comment (zoldar) [inline lib/cloudini/url.ex]: Please consider using:

http://elixir-lang.org/docs/stable/elixir/Path.html#extname/1

Comment (TisButMe) [inline lib/cloudini/url.ex]: ack.

**Comment by TisButMe** [comment]
@zoldar Updated.

**Comment by aerosol** [comment]
1.1.0 published to hex

**Pull request merged**